scripts/ci/copyright_check.py

```diff
@@ -28,6 +28,17 @@ def get_header(has_shebang, regexes):
     indices = range(init_idx, init_idx + len(regexes))
     return zip(regexes, indices)
 
+
+def match_somewhere(regexes, lines):
```

Comment:
Can you please add a brief description?

Reply:
Fixed with df6e3a0244c78c2376c35d27e9714834077b9db0